cis/plugins/validation/mozilliansorg_publisher_plugin.py

```diff
@@ -5,6 +5,10 @@
 utils.StructuredLogger(name=__name__, level=logging.INFO)
 logger = logging.getLogger(__name__)
 
+# Allows user creation by this publisher.
+CAN_CREATE_USER = True
+ENFORCE_ATTR_WHITELIST = False
+
 
 def run(publisher, user, profile_json):
```

Comment:
we probably should call `user` something like `user_from_vault` (i got confused even thus i wrote the contents of the plugin initially)